metrics-operator/api/v1alpha3/keptnmetric_types.go

```diff
@@ -51,6 +53,12 @@ type ProviderRef struct {
 	Name string `json:"name"`
 }
 
+// RangeSpec defines the time range for which data is to be queried
+type RangeSpec struct {
+	// +kubebuilder:default:="5m"
+	Interval string `json:"interval,omitempty"`
```

Comment:
please also add a comment describing this property to support automatic doc generation for that one